config/routes.rb

```diff
@@ -4,7 +4,16 @@
   # Defines the root path route ("/")
   # root "articles#index"
```

Comment:
- [ ] [OPTIONAL] As it is just one issue I am assigning it as optional, please add a root or default route so the user is presented with a default view most probably the users#index route, currently it is shoeing the default rails view, which is not appropriate from UX point of view :+1:

Reply:
Thank you for your feedback @PinkMoon25